code/modules/power/cell.dm

```diff
@@ -204,14 +204,23 @@
 	materials = list(MAT_GLASS=60)
 	rating = 3
 	chargerate = 1500
+	
+/obj/item/weapon/stock_parts/cell/guncrafthigh  //Battery variants for guncrafting
+	name = "high-capacity gun power cell"
+	origin_tech = "powerstorage=2"
+	icon_state = "hcell"
+	maxcharge = 10000

```

Comment:
Starting from high-power, of course, since for some reason from base to high it ramps up from 1000 to 10000.